frontend/pages/index.js

```diff
@@ -180,6 +182,62 @@ const Homepage = () => {
       setBackgroundLoaded3840(true);
     };
   }, []);
+
+  const MilestoneBox = ({ headingText, description }) => {
+    return (
+      <Box
+        w={["150px", "180px", "300px", "300px", "400px", "500px"]}
+        // bg="gray.100"
+        py={5}
+      >
+        <Text
+          fontSize={"xl"}
+          ml={5}
+          color="orange.500"
+          fontWeight="bold"
+          textAlign="center"
+        >
+          {headingText}
+        </Text>
+        <Text textAlign="center">{description}</Text>
+      </Box>
+    );
+  };
+
+  const JoinDiscordButton = () => {
```

Comment:
If you want to have reusable button styling then define it it in theme/button under a new `discordButton` variant, that way we will be able to reuse styling across other pages and components